interface/modules/zend_modules/vendor/dompdf/dompdf/www/demo.php

```diff
@@ -8,8 +8,7 @@
 
 if ( isset( $_POST["html"] ) && $is_local ) {
 
-  if ( get_magic_quotes_gpc() )
-    $_POST["html"] = stripslashes($_POST["html"]);
+  $_POST["html"] = strip_escape_custom($_POST["html"]);
```

Comment:
I will revert this change (and I will remove this file; no reason to have demo stuff around in the codebase).